envoy/type/v2/BUILD

```diff
@@ -0,0 +1,14 @@
+load("//bazel:api_build_system.bzl", "api_proto_library", "api_go_proto_library")
+
+licenses(["notice"])  # Apache 2
+
+api_proto_library(
+    name = "range",
+    srcs = ["range.proto"],
+    visibility = ["//visibility:public"],
```

Comment:
What should the visibility be set to? 

//envoy/api/v2:friends gives the error:
ERROR: /source/envoy/api/v2/route/BUILD:5:1: Target '//envoy/type/v2:range' is not visible from target '//envoy/api/v2/route:route'. Check the visibility declaration of the former target if you think the dependency is legitimate

Reply:
Just leave it empty, the default in https://github.com/envoyproxy/data-plane-api/blob/master/bazel/api_build_system.bzl#L81 should apply unless you have specific need to restrict visibility as done elsewhere.

Reply:
Empty does not work,  default => private, I get the same error as mentioned above. This type is referenced in envoy/api/v2/route